JapaneseFilter/sections/specific.txt

```diff
@@ -3,6 +3,38 @@
 !
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
+||joins.com/HomeAds/$subdocument
+||p.net-public.com/js/b.js
+||seesaa.net/image/loopad*$subdocument,third-party
+||vippers.jp/include*$xmlhttprequest
+||vippers.jp/rss.html
+47news.jp##.billbord-js
+amaebi.co##.advertisement-position
+amaebi.co##.blogroll-wrapper
+amaebi.co##.side--right > div:last-child
+asahi.com##.p-ad--topNews
+asahi.com##.p-chumoku
+asahi.com##.p-rectangleAd
+ascii.jp###artAds
+ascii.jp##.lnkBanner
+ascii.jp##.pickwrap
+chunichi.co.jp,tokyo-np.co.jp##.pr
+j-cast.com##.chumokujoho
+japan.cnet.com###ad_textbottom
+japan.cnet.com###content-bottom-block
+japanese.joins.com##.YahooAds
+jiji.com##.ArticleInnerAD
+live.nicovideo.jp,live2.nicovideo.jp##[class^=___billboard-ad]
```

Comment:
Could you please add `"` (after this part `class^=`)  to this rule?
I mean:
`live.nicovideo.jp,live2.nicovideo.jp##[class^="___billboard-ad"]`

Reply:
Done.